src/singUp/components/screenSignUp.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View } from 'react-native';
+import Header from './header';
+import AlbumList from './albumList';
+import UselessTextInput from './textBox';
+import Texts from './texts';
+import Buton from './buton';
+
+const ScreenSign = () =>{
```

Comment:
add space after the arrow
```suggestion
const ScreenSign = () => {
```